main.c

```diff
@@ -160,22 +160,28 @@ int WINAPI _tWinMain (HINSTANCE hThisInstance,
   PrintDebug(_T("Shell32.dll version: 0x%lx"), shell32_version);
 #endif
 
-
   /* Check if a previous instance is already running. */
   if ((FindWindow (szClassName, NULL)) != NULL)
     {
         /* GUI already running */
-        ShowLocalizedMsg(IDS_ERR_GUI_ALREADY_RUNNING);
-        exit(1);
+        already_running = 1;
+    }
+  else
+    {
+        UpdateRegistry(); /* Checks version change and update keys/values */
     }
-
-  UpdateRegistry(); /* Checks version change and update keys/values */
 
   GetRegistryKeys();
+
+  EnsureDirExists(o.config_dir);
   /* Parse command-line options */
   ProcessCommandLine(&o, GetCommandLine());
 
-  EnsureDirExists(o.config_dir);
```

Comment:
Moving this up is problematic in normal use as o.config_dir can change after ProcessCommandLine. Anyway existence of o.config_dir is ensured in ImportConfigFile() so keep the original order.